<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Writer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pic.jpg</normaloff>pic.jpg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>723</width>
        <height>10000</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10000</height>
       </size>
      </property>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>1</y>
         <width>711</width>
         <height>10000</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10000</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>方正正黑简体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>方正正黑简体</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回（完成写作）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>方正正黑简体</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="text">
        <string>全部句子</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>方正正黑简体</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开头句</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>方正正黑简体</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="text">
        <string>过渡句</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>方正正黑简体</family>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="text">
        <string>结尾句</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
